packages/insomnia-inso/src/db/index.ts

```diff
@@ -36,36 +35,33 @@ export const emptyDb = (): Database => ({
 
 export type DbAdapter = (
   dir: string,
-  filterTypes?: Array<$Keys<Database>>,
+  filterTypes?: (keyof Database)[],
 ) => Promise<Database | null>;
 
 type Options = {
-  workingDir?: string,
-  appDataDir?: string,
-  filterTypes?: Array<$Keys<Database>>,
+  workingDir?: string;
```

Comment:
This might be correct but i was thrown off by the `;` at the end of these lines...Were they supposed to be changed from `,`s?

Reply:
yep, prettier does this.  it's very strongly the standard in typescript (one difference from flow).  both are allowed.